Node 19 fails World Parameter invalid JSON test

I checked out main, ran the test suite and got an error on one test:
```
1) Scenario: Invalid JSON # features\world_parameters.feature:15
   √ Before # features\support\hooks.ts:14
   √ Before # features\support\hooks.ts:18
   √ Given a file named "features/passing_steps.feature" with: # features\step_definitions\file_steps.ts:11
       """
       Feature: a feature
         Scenario: a scenario
           Given the world parameters are correct
       """
   √ When I run cucumber-js with `--world-parameters '{"a":}'` # features\step_definitions\cli_steps.ts:22
   √ Then it fails # features\step_definitions\cli_steps.ts:81
   × And the error output contains the text: # features\step_definitions\cli_steps.ts:141
       """
       --world-parameters passed invalid JSON: Unexpected token }
       """
       AssertionError: expected 'This Node.js version (v19.4.0) has no…' to include '--world-parameters passed invalid JSO…'
           at World.<anonymous> (C:\Users\tendo\Projects\gitHub\cucumber-js\features\step_definitions\cli_steps.ts:146:29)
   - And the error output contains the text: # features\step_definitions\cli_steps.ts:141
       """
       {"a":}
       """
   √ After # features\support\hooks.ts:55
```

Likely there's been a change in the error string in 19 and this will need to be detected and compensated for in the verification process.
----
 
*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/ISSUE_TEMPLATE/bug_report.md)*

--- a/features/world_parameters.feature
+++ b/features/world_parameters.feature
@@ -17,7 +17,7 @@ Feature: World Parameters
     Then it fails
     And the error output contains the text:
       """
-      --world-parameters passed invalid JSON: Unexpected token }
+      --world-parameters passed invalid JSON: Unexpected token
       """
     And the error output contains the text:
       """

